translations/strings.yaml

```diff
@@ -82,3 +98,172 @@ splittunnel:
     comment:
       - Displayed in notification after the user added a missing app to the split-tunnel List; %1 is the name of the added application
 
+  excellentLabel: Excellent
+  #Screen 843
+  selectionHeader: Give feedback
+  reviewHeader: Give us 5 stars!
+  reviewBody: If you enjoy using Mozilla VPN, would you review it? Thanks for your support!
+  leaveReviewButton: Leave a Review
+  skipLink: Skip
+
+#Additional messaging for network issues with split tunneling settings
+splitTunnelNetworkMessaging:
+  infoText: For any questions about setting app permissions, we've provided some helpful tips on our website.
+  infoLinkText: Learn more
+  
+#adding multi-hop feature
+multiHopFeature:
+  #Screen MacOS/134
+  singleToggleHeader: Single-hop
+  multiToggleHeader: Multi-hop
+  #Screen MacOS/140
+  cardHeader: What is multi-hop VPN?
+  cardBody: Multi-hop VPN routes your traffic through two servers instead of one for extra security and privacy. This may slow down your connection.
+  entryLocationHeader: Select entry location
+  exitLocationHeader: Select exit location
+  infoText: Exit location represents your main VPN server.
+  #Screen MacOS/136
+  pageHeader: Select entry location
+  #Screen MacOS/145
+  pageHeader: Select exit location
+  #Screen MacOS/142
+  connectionsHeader: Recent connections
+  arrowAltText: To
```

Comment:
Please provide a comment explaining that the string is used as alternative text for the image of an arrow (connecting what?)